profile_test.go

```diff
@@ -128,8 +128,28 @@ func TestProfile_MarshalCBOR_URL_OK(t *testing.T) {
 	assert.Equal(t, expected, actual)
 }
 
-// TestProfile_MarshalCBOR_URL_NOK tests the invalid case of CBOR marshaling of profile been set as an URI
-func TestProfile_MarshalCBOR_URL_NOK(t *testing.T) {
+// TestProfile_UnmarshalCBOR_URL_OK tests the valid case of CBOR Unmarshaling of profile claim set as URL
+func TestProfile_UnmarshalCBOR_URL_OK(t *testing.T) {
+	inputURL := "http://example.com"
+	profile, err := NewProfile(inputURL)
+	assert.Nil(t, err)
```

Comment:
this is technically a precondition, which should be a test requirement rather than a test assertion:
```suggestion
	require.assert.Nil(t, err)
```

Reply:
Can consider this enhancement as part of the Test Enhancement Issue #24